src/PackagePrivate/Humanizer/Strategy/EnglishStrategy.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace EDTF\PackagePrivate\Humanizer\Strategy;
+
+class EnglishStrategy implements LanguageStrategy
+{
+    public function applyOrdinalEnding(int $number): string
+    {
+        if ( $number % 100 >= 11 && $number % 100 <= 13 ) {
+            return $number. 'th';
+        }
+
+        return $number . [ 'th','st','nd','rd','th','th','th','th','th','th' ][$number % 10];
+    }
+
+    public function composeFullDateString(string $year, string $month, string $day): string
+    {
+        return $month . ' ' . $day . ', ' . $year;
+    }
```

Comment:
I like the strategy approach for applyOrdinalEnding. Not sure why it is needed for composeFullDateString. It looks like you could do that with standard messages. Or will there be more complexity later on?

Reply:
No problem at all. We can handle this via message template. Probably, it's even better. Will fix this in next PR